docs/UserGuide.md

```diff
@@ -108,6 +108,101 @@ You should notice the GUI of the application pop up.
 
 --------------------------------------------------------------------------------------------------------------------
 
+# Introducing LinkMeIn: A Quick Tour #
+_If you are new to LinkMeIn, welcome aboard! We know that navigating the world of internship applications can be
+daunting, especially when you have multiple opportunities in sight. But fret not, because with LinkMeIn,
+you have a reliable companion to guide you through this exciting journey._
+
+This tour aims to help you get started with LinkMeIn by introducing you to its essential features that you will be
+using quite often. In this tour, you will be learning how and when to:
+- [add a company](),
+- [view its details](),
+- [edit its details]()
+- [delete it]().
+
+Before we embark on this tour remember to download LinkMeIn and have your computer set up correctly. If you have not
+done so, do check out the [Quick Start Guide](#quick-start) for more information.
+Now, follow us along this tour and transform yourself into a master of your internship application!
+
+## 1. Adding your first company ##
+Adding a company is the first step to tracking your internship applications. Let us start off by adding your
+first company to LinkMeIn.
+
+1.Whether you are scrolling through your LinkedIn feed or browsing through a job portal, if you come across a
+company that you are interested in, you can add it to LinkMeIn. Let's say you are interested in a Software Engineering
+role at TikTok.
+
+2.Type the following `add c/Tiktok r/Software Engineer s/PA n/John Tan d/10-10-2023 e/johntan@example.com
+p/987654321 pr/high` in the command box and press Enter.
+
+<div class="typography-box note">
+  <span class="icon">ℹ️</span>
+  <span class="content">
+    <strong>Note: If you wish to know more about what each parameter means and how to use the add command, you can do so <a href="#adding-a-company--add-">here</a>.</strong>
```

Comment:
```suggestion
    <strong>Note: If you wish to know more about what each parameter means and how to use the add command, you can do so <a href="#adding-a-company-add">here</a>.</strong>
```
hyperlink dont seem to work, may be try this instead

Reply:
I tried on my side, it works tho